src/content/en/2020/cms.md

```diff
@@ -23,3 +22,364 @@ published: 2020-12-09T00:00:00.000Z
+  image="cms-pwa-compliance-lighthouse-score.png",
+  caption="PWA compliance.",
+  description="Bar chart showing the median PWA compliance score for each of the top 5 cmss. Wordpress has a median score of 32% and the rest (Joomla, Drupal, Wix and Squarespace) all have 29%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=904129607&format=interactive",
+  sheets_gid="1497775920",
+  sql_file="median_lighthouse_score.sql"
+  )
+}}
+
+The median Lighthouse PWA score is 29% for the top 5 CMSs, with the exception of WordPress which comes in at 32%. Across all CMSs, the average median score is 34%, with a minimum of 4% and a maximum of 71%.
+
+**Accessibility**
+
+An accessible website is a site designed and developed so that people with disabilities can use them; web accessibility also benefits people without disabilities, such as those on slow internet connections. [A full discussion can be seen here](https://www.w3.org/WAI/fundamentals/accessibility-intro/#what). Lighthouse provides a set of accessibility audits and it returns a weighted average of all of them (see [Scoring Details](https://docs.google.com/spreadsheets/d/1Cxzhy5ecqJCucdf1M0iOzM8mIxNc7mmx107o5nj38Eo/edit#gid=1567011065) for a full list of how each audit is weighted).
+
+Each accessibility audit is pass or fail, but unlike other Lighthouse audits, a page doesn't get points for partially passing an accessibility audit. For example, if some elements have screen reader-friendly names, but others don't, that page gets a 0 for the screen reader-friendly-names audit.
+
+{{ figure_markup(
+  image="cms-accessibility-compliance-lighthouse-score.png",
+  caption="Accessibility compliance.",
+  description="Bar chart showing the median Accessibility compliance score for each of the top 5 cmss. Wordpress has a median score of 84%, Joomla 80%, Drupal 84%, Wiz is leading with 94% and Squarespace has 90%",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=1331234903&format=interactive",
+  sheets_gid="1497775920",
+  sql_file="median_lighthouse_score.sql"
+  )
+}}
+
+The median Lighthouse accessibility score for the top 5 CMSs is all above 80%. Across all CMSs, the average median Lighthouse score is 78%, with a minimum of 44% and a maximum of 98%. We thus see that the top 5 CMSs are better than average, with some better than others. Wix and Squarespace have the highest scores of the top 5. Possibly these platforms being proprietary helps here. 
+
+The bar should be higher here, though. An average score of 78% across all CMSs still leaves significant room for improvement, and the maximum score of 98% shows even the “best” CMS for accessibility compliance has room for improvement. Improving accessibility is essential and urgent work.
+
+
+## What’s the impact of CMSs on the environment?
```

Comment:
Suggest renaming the heading. But since you reference the question later, I'd also suggest asking it in the introductory paragraph of the section.

```suggestion
## Environmental impact
```